### PR TITLE
[ls-qpack] Update to latest version and fix build error

### PR DESCRIPTION
**Pull request by JonLiu1993**
[ls-qpack] Update to latest version and fix build error

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version.~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] When updating the upstream version, the `"port-version"` is reset (removed from `vcpkg.json`).
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

Usage tested successfully by ls-qpack:x64-windows:
````
ls-qpack provides CMake targets:

  # this is heuristically generated, and may not be correct
  find_package(ls-qpack CONFIG REQUIRED)
  target_link_libraries(main PRIVATE ls-qpack::ls-qpack)
````

### CODE REVIEW

File: ports/ls-qpack/portfile.cmake [hunk 1]
```diff
@@ -6,11 +6,8 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO litespeedtech/ls-qpack
     REF "v${VERSION}"
-    SHA512 7677f673b4b23a68ad5e899706f17536777b30d7e91c63d3ea97504a6a2885cf7f431c191ac0581631723151050f914ec31bcb84e2b6e3fcdf4140cde0a18063
+    SHA512 74d4b2ea96bf0de43009cc121c8e57caff83be53c613236b01dce4ac4c12505d0d9fec07d9152ca62166947a160de2ab3f7bf19fb203a60b44507516a927ecb8
     HEAD_REF master
-    PATCHES
-        xxhash.patch
-        cmake-export.patch
```
Comment (JonLiu1993) [inline ports/ls-qpack/portfile.cmake:13]: These patches have been applied to the latest version

**Comment by talregev** [comment]
ls-qpack now use official cmake export:
https://github.com/litespeedtech/ls-qpack/pull/59

File: ports/ls-qpack/portfile.cmake (hunk 1 above)
Comment (talregev) [reply ports/ls-qpack/portfile.cmake:13]: Yes, I contribute to upstream.

**Comment by JonLiu1993** [comment]
> ls-qpack now use official cmake export: [litespeedtech/ls-qpack#59](https://github.com/litespeedtech/ls-qpack/pull/59)
> 
> you should change to official cmake export, and also change msquic as well that use unofficial ls-qpack cmake export.

@talregev , Thanks for your reminder. The latest version uses the official cmake export. Some ports that depend on it have not been modified yet. I will fix them one by one according to the CI test results.

**Comment by talregev** [comment]
> > ls-qpack now use official cmake export: [litespeedtech/ls-qpack#59](https://github.com/litespeedtech/ls-qpack/pull/59)
> > you should change to official cmake export, and also change msquic as well that use unofficial ls-qpack cmake export.
> 
> @talregev , Thanks for your reminder. The latest version uses the official cmake export. Some ports that depend on it have not been modified yet. I will fix them one by one according to the CI test results.

Thank you for your work and your help!
Because I added this port for msh3, that this port currently is the only one that use it.
This will save you time to wait for ci to fail.
https://github.com/microsoft/vcpkg/blob/master/ports/msh3/dependencies_fix.patch#L10


We need to modify msh3 port and not msquic as I say in the previous msg.

**Comment by talregev** [comment]
@JonLiu1993 msh3 just release a new version 0.7.0
https://github.com/nibanks/msh3/releases

**Pull request merged**